src/applications/ezr/tests/unit/utils/helpers/file-attachments.unit.spec.js

```diff
@@ -0,0 +1,46 @@
+import { expect } from 'chai';
```

Comment:
Would something more like this be clearer and a little simpler? I DRY'd up some of the variables and removed the before blocks: 
```suggestion
import { expect } from 'chai';
import {
  createPayload,
  parseResponse,
} from '../../../../utils/helpers/file-attachments';

describe('file-attachments', () => {
  const mockFile = new File(['test'], 'test.txt', { type: 'text/plain' });

  describe('ezr `createPayload` method', () => {
    it('should create a FormData instance with file', () => {
      const payload = createPayload(mockFile);
      expect(payload.get('form1010_ezr_attachment[file_data]')).to.equal(
        mockFile,
      );
    });
  });

  describe('ezr `parseResponse` method', () => {
    it('should return an object with the name, confirmation code and file size', () => {
      const response = {
        data: {
          attributes: {
            guid: 'test-guid',
          },
          id: 'test-id',
        },
      };

      const mockResult = parseResponse(response, mockFile);

      expect(mockResult).to.deep.equal({
        name: 'test.txt',
        confirmationCode: 'test-guid',
        size: 4,
      });
    });
  });
});

```

Reply:
Hey, thank you for cleaning this up. Just an FYI, I pulled all of this code from the EZ and essentially made it fit for the EZR. Maybe we need to look at cleaning the EZ tests up as well?

Reply:
Yeah looks like they are exactly the same except for spec names so it would make sense to make them consistent. If you want to update it as part of this PR I'd say go for it, but also if you don't want extra scope you can hold off.

Reply:
All right, sweet. It's not a problem, I can make those changes in this PR. I'll update the description to include the EZ as well.

Reply:
For the sake of keeping things separate. Lets not make changes to another app in this PR. Yes, it may be the same change, but it should be in its own PR.

Reply:
Okay, I'll create a separate PR for that update @longmd !